src/components/box/DateRangeConditionBox.tsx

```diff
@@ -0,0 +1,40 @@
+import DownloadConditionBox from "./DownloadConditionBox";
+import {
+  DateRangeCondition,
+  DownloadConditionType,
+} from "../../pages/detail-page/context/DownloadDefinitions";
+import React, { useMemo } from "react";
+import { Typography } from "@mui/material";
+
+interface DateRangeConditionBoxProps {
+  dateRangeCondition: DateRangeCondition;
+}
+
+const DateRangeConditionBox: React.FC<DateRangeConditionBoxProps> = ({
```

Comment:
Try this http://localhost:5173/details?uuid=06b09398-d3d0-47dc-a54a-a745319fbece&tab=abstract

I cannot move the dot and select time range, end date is 10-1800